### DISCUSSION TITLE
exception/try-with-resources

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#240] [Announcements]
# exception/try-with-resources

本文详细介绍了 Java 中 try-with-resources 语句的作用和用法，阐述了其如何优雅地处理资源的关闭与异常处理。同时，文章还提供了 try-with-resources 的实际应用示例和与传统 try-catch-finally 的对比分析。阅读本文，将帮助您更深入地了解 Java 中的 try-with-resources 语句，有效提升资源管理的编程技巧。

https://javabetter.cn/exception/try-with-resources.html

**Comment by Wuwanwan** [comment]
始终有限考虑使用 try-with-resources，而不是 try–catch-finally。
这句中有个错别字，“有限”应该改为“优先”
始终优先考虑使用 try-with-resources，而不是 try–catch-finally。
